.github/workflows/bump_version.yml

```diff
@@ -0,0 +1,63 @@
+name: Bump Version
+
+on:
+  workflow_dispatch:
+    inputs:
+      newVersion:
+        description: "Version name"
+        required: true
+        type: string
+
+jobs:
+  bump_version:
+    name: Bump version
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the latest code
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Bump version script
+        id: set_version
+        run: |
+          perl -i -pe 's/^(templateScriptVersion=(.*))$/"templateScriptVersion=${{ github.event.inputs.newVersion }}"/e' version.properties
+
+      - name: Bump Version XML
+        uses: chkfung/android-version-actions@v1.2.1
+        with:
+          gradlePath: sample-xml/app/build.gradle.kts
+          versionCode: ${{ github.run_number }}
```

Comment:
@doannimble We will not change the `versionCode`, right? 🙇

Reply:
@Tuubz We will, please take a look at POW here: https://github.com/nimblehq/android-templates/pull/386, it will be automatically increase by `1` 🙏

Reply:
@doannimble Is there any reason we want to increase the version code for the samples? [We didn't do this before](https://github.com/nimblehq/android-templates/pull/381/files) 💭 

Also, I think we should have two files (we're missing one file):

- `.github/workflows/bump_version.yml` (used to bump version for samples)
- `.cicdtemplate/.github/workflows/bump_version.yml` (used to bump version for new projects)

What do you think? 🙏 

cc @luongvo

Reply:
@Tuubz Oh, my mistake, we should increase the template, not the sample. I will correct it, but increasing the version code is necessary IMO 🙏

Reply:
@doannimble No, we don't need to increase the template. 

The `versionCode` and `versionName` will always be fixed for `template`

Because if a developer wants to generate a new project, they want to start from the beginning

Anyways, would you mind explaining why it's necessary to increase the version code for samples? 🙇

Reply:
@Tuubz I got your point now. It is only the sample code, not for product release, so increasing the `versionCode` is unnecessary.

Reply:
@doannimble One more thing, shall we also add `.cicdtemplate/.github/workflows/bump_version.yml`?

Reply:
@Tuubz Nope, we shalln't. Since we need to input the version code and control the path manually. Besides, we don't use it for the production release 🙏

Reply:
@doannimble I think we should, for the generated project too, the default path of the gradle file should be `app/build.gradle.kts`. Let's say we generate and push the generated project into Github again, we will need to have that workflow beforehand.

We're missing that workflow in Flutter templates, I will create a chore task to add the missing one 🙏